src/Paket.Core/ProjectFile.fs

```diff
@@ -941,6 +941,9 @@ module ProjectFile =
                 project.ProjectNode.AppendChild analyzersNode |> ignore
             )
 
+    let touch (project:ProjectFile) =
+        if File.Exists(project.FileName) then
```

Comment:
On second thought - maybe it would make more sense to slightly change the behavior of `save` instead of adding touch, along the lines of:
```
   let save forceTouch project =
        if Utils.normalizeXml project.Document <> project.OriginalText then 
            verbosefn "Project %s changed" project.FileName
            project.Document.Save(project.FileName)
        elif forceTouch && File.Exists(project.FileName) then
            File.SetLastWriteTimeUtc(project.FileName, DateTime.UtcNow)
```